src/status_im/extensions/core.cljs

```diff
@@ -474,6 +479,10 @@
                 {:permissions [:read]
                  :value       :extensions.chat.command/send-message
                  :arguments   {:params :map}}
+                'chat.command/start-public-chat
```

Comment:
How about `chat/open-public-chat`  ? And how about `navigate-to?` instead of `open`?

Reply:
always the same problem: symbol naming 🤣